### ISSUE TITLE
Containerd version reported as unknown when listing nodes in cluster

### ISSUE THREAD

**Issue by dnoland1** [rancher/k3s#159]
Details in https://github.com/rancher/rancher/issues/18593

**Comment by erikwilson** [comment]
This should be fixed with #626 and available for testing in https://github.com/rancher/k3s/releases/tag/v0.7.0-rc5. Thanks for reporting @dnoland1 !

**Comment by galal-hussein** [comment]
K3s Version: v0.7.0-rc5

I was able to verify that the containerd version is displayed correctly both in cli and as imported in Rancher
```
k3s crictl version
Version:  0.1.0
RuntimeName:  containerd
RuntimeVersion:  v1.2.7-k3s1
RuntimeApiVersion:  v1alpha2
```

![containerd-version](https://user-images.githubusercontent.com/3309968/61090581-7b4c3e80-a43f-11e9-8609-842506785ba2.png)
